packit/api.py

```diff
@@ -739,5 +737,4 @@ def push_updates(self, update_alias: Optional[str] = None):
 
     def clean(self):
         """ clean up stuff once all the work is done """
-        if self._up:
-            self.up.command_handler.clean()
+        logger.info("cleanup (there are no objects to clean)")
```

Comment:
> there are no objects to clean

You seem pretty confident, so I won't ask, maybe just:

- `info` -> `debug`
-  `cleanup` -> `PackitAPI.cleanup`

Reply:
this is related to the issue with pods not being cleaned up: I had to change the code so the pod is cleaned after the command finishes, hence we have nothing to clean any more.